src/Deployer.php

```diff
@@ -134,14 +140,14 @@ public function getOutput()
     /**
      * @param string $name
      * @return mixed
-     * @throws \InvalidArgumentException
+     * @throws InvalidArgumentException
```

Comment:
@gordalina I don't understand why you'd want to make phpdoc hard to read? The whole point of phpdoc is for people to read, not for silly IDEs.